server/views/pages/prisonerList/index.njk

```diff
@@ -18,11 +18,8 @@
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-full">
       <h1 class="govuk-heading-l">Manage learning and work progress</h1>
-
-      <form class="form" method="get" novalidate="">
```

Comment:
I've moved the form back into `searchBox` - see the comments in that file to see why